homeassistant/components/alexa/capabilities.py

```diff
@@ -86,28 +88,23 @@ def name(self) -> str:
         """Return the Alexa API name of this interface."""
         raise NotImplementedError
 
-    @staticmethod
-    def properties_supported() -> list[dict]:
```

Comment:
Why change this?

Reply:
These static methods are overwritten with normal instance methods. Which means technically the signatures aren't compatible. You can't, for example, just exchange a call to `AlexaCapability.properties_supported` with one to a subclass `AlexaEndpointHealth.properties_supported`.
https://github.com/home-assistant/core/blob/70648d6e3b42bb0910e33e5c6a4e1e8c0789d82c/homeassistant/components/alexa/capabilities.py#L376-L378

Furthermore, they are accessed via `self.xxx` anyway. So removing `staticmethod` here seems fine.
https://github.com/home-assistant/core/blob/70648d6e3b42bb0910e33e5c6a4e1e8c0789d82c/homeassistant/components/alexa/capabilities.py#L190-L196